packages/kernel/source/services/config/drivers/local.ts

```diff
@@ -112,6 +112,7 @@ export class LocalConfigLoader implements Contracts.Kernel.ConfigLoader {
 		this.validationService.validate(
 			this.#loadFromLocation(["validators.json"]),
 			Joi.object({
+				keystore: Joi.string().optional(),
 				secrets: Joi.array().items(Joi.string()).optional(),
```

Comment:
Since we are always setting secrets, can it be set as required?  

```suggestion
				secrets: Joi.array().items(Joi.string()).required(),
```